R/Spectra-functions.R

```diff
@@ -39,14 +39,17 @@ addProcessing <- function(object, FUN, ...) {
 #' @param msLevel optional `integer` defining the MS level(s) to which the
 #'     function should be applied.
 #'
+#' @param ... required. Support for passing additional parameters without
+#'     throwing an error.
+#'
 #' @return `matrix` with columns `"mz"` and `"intensity"`.
 #'
 #' @importMethodsFrom IRanges extractList replaceROWS
 #'
 #' @importClassesFrom IRanges IRanges
 #'
 #' @noRd
-.remove_peaks <- function(x, spectrumMsLevel, centroided = NA, t = "min",
+.peaks_remove <- function(x, spectrumMsLevel, centroided = NA, t = "min",
```

Comment:
I completely understand you @sgibb . Note however that the naming scheme I used with `.peaks_remove` was not to change it to `type_verb`, but rather to have all internal functions that are applied to peak matrix data to start with `.peaks_`: the internal function of the `bin` method, is called now `.peaks_bin` and the one for `clean` `.peaks_clean`. 

This was my rationale, I wanted to quickly identify the functions that can be applied to peaks matrix - also for autocompletion, you simply type `.peaks_` and it will list you all possible functions.

Reply:
I've moved the functions to a new file (peaks-functions.R) and wrote a small description about the naming scheme.